### ISSUE TITLE
Symfony Package Conflicts

### ISSUE THREAD

**Issue by PHLAK** [stecman/composer-bash-completion-plugin#19]
Thanks for [enabling Composer 2 support](https://github.com/stecman/composer-bash-completion-plugin/issues/18) so quickly. However, now I'm getting conflicts with other packages due to the required `symfony/console` version.

```
$ composer global require stecman/composer-bash-completion-plugin dev-master
Changed current directory to /home/chris/.config/composer
./composer.json has been updated
Running composer update stecman/composer-bash-completion-plugin
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - stecman/symfony-console-completion 0.7.0 requires symfony/console ~2.3 || ~3.0 -> found symfony/console[v2.3.0, ..., v2.8.52, v3.0.0, ..., v3.4.45] but the package is fixed to v5.1.7 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - stecman/composer-bash-completion-plugin dev-master requires stecman/symfony-console-completion ~0.7.0 -> satisfiable by stecman/symfony-console-completion[0.7.0].
    - Root composer.json requires stecman/composer-bash-completion-plugin dev-master -> satisfiable by stecman/composer-bash-completion-plugin[dev-master].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I believe this is due to the `"stecman/symfony-console-completion": "~0.7.0"` dependency. It looks like you have an [updated version](https://github.com/stecman/symfony-console-completion/releases/tag/0.11.0) that supports newer Symfony package versions though. I'm not sure but updating this dependency might be all you need to do to fix that.

**Comment by stecman** [comment]
Fixed in 14b1ef54ff377e392d23dbebfe25471915b34cc6. Also closes #20 :+1:

**Comment by PHLAK** [comment]
Thank you!
